src/go/rpk/pkg/api/admin/api_broker.go

```diff
@@ -40,19 +45,20 @@ type Broker struct {
 
 // Brokers queries one of the client's hosts and returns the list of brokers.
 func (a *AdminAPI) Brokers() ([]Broker, error) {
```

Comment:
It seems we don't return `version` and lose this information. We should return it to the users otherwise they may be confused because `/v1/brokers` and `/v1/cluster_view` return the node's local view of the cluster. And when user queries info several times we choose a node to query randomly so there is a chance that we query a node with obsolete information. Without the version a user can't catch this situation and it may cause problems.

Reply:
Some context

> These membership changes were difficult to execute correctly in unhealthy networks. Both rpk cluster info and the /v1/brokers API could return arbitrarily stale views of the cluster, which could lead to operators concluding that a node had completed its decommissioning process when it was, in fact, still handing off data to other members.

http://jepsen.io/analyses/redpanda-21.10.1

Reply:
Thanks, I changed this method to ClusterView and now is returning the cluster view version and we print it in both places where we use this:

- `rpk cluster maintenance status`
- `rpk redpanda admin brokers list`